odl/operator/default_ops.py

```diff
@@ -877,9 +887,12 @@ def _call(self, x, out=None):
         rn(3).element([1.0, 2.0, 3.0])
         """
         if out is None:
-            return self.range.element(self.vector.copy())
+            if isinstance(self.__vector, float):
+                return self.range.element(self.__vector)
+            else:
+                return self.range.element(self.__vector.copy())
```

Comment:
I'm not sure why copy was/is used here, but when using it with a `float` it crashes. So I made a quick-fix. Comments?